submissions/kasionio/oop-exercise/index.js

```diff
@@ -0,0 +1,100 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/kasionio/a-tiny-JS-world/tree/gh-pages
+   Web app: https://kasionio.github.io/a-tiny-JS-world
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, legs, saying, friends = 'forever alone') {
```

Comment:
Neither of immediate child classes of `Inhabitant` ever pass friends via `super`. Is it a redundant param?

Reply:
Define friends property as an empty array inside the class